.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+name: Pull Request
+
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
Should we add something like:
```
    branches:
      - main
```

Reply:
Is there a reason to explicitly specify ```types: [opened, synchronize]```?

Reply:
> Is there a reason to explicitly specify `types: [opened, synchronize]`?

Based on my previous experience, I faced a problem that when the pull request has been opened, all jobs was ran as expected. But when already opened pull request gets updated, the jobs are not re-ran for this update, and because of that now I explicitly add ```types: [opened, synchronize]```

Reply:
> Should we add something like:
> 
> ```
>     branches:
>       - main
> ```

done